provisioner/secret.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var base = require('taskcluster-base');
+var assert = require('assert');
+var debug = require('debug')('aws-provisioner:Secret')
+var taskcluster = require('taskcluster-client');
+
+var Secret = base.Entity.configure({
+  version: 1,
+
+  // TODO: verify these are valid azure keys - heeeelp please Jonas
+  partitionKey: base.Entity.keys.StringKey('token'),
+  rowKey: base.Entity.keys.StringKey('provisionerId'),
+
+  properties: {
+    token: base.Entity.types.String,
+    created: base.Entity.types.Date,
+    provisionerId: base.Entity.types.String,
+    workerType: base.Entity.types.String,
+    secrets: base.Entity.types.JSON,
+  },
```

Comment:
Just realized we need to add a property `scopes` too.. as in the scopes that should be used for the temporary credentials that will be issued when you call to get these secrets.